api/src/main/java/org/pmiops/workbench/api/AuditController.java

```diff
@@ -0,0 +1,138 @@
+
+  private static final Logger log = Logger.getLogger(AuditController.class.getName());
+  private static final String AUDIT_SINK_NAME = "cdr_audit_logs";
+  // How many days into the past (including today) logs should be checked. This could become a
+  // request parameter if the need arises.
+  private static final int AUDIT_DAY_RANGE = 7;
+  // BigQuery log sink table names are have a suffix like "20170103", per
+  // https://cloud.google.com/logging/docs/export/using_exported_logs#table_organization
+  private static final DateTimeFormatter auditTableNameDateFormatter =
+      new DateTimeFormatterBuilder()
+        .appendValue(ChronoField.YEAR, 4)
+        .appendValue(ChronoField.MONTH_OF_YEAR, 2)
+        .appendValue(ChronoField.DAY_OF_MONTH, 2)
+        .toFormatter();
+
+  private final Clock clock;
+  private final BigQueryService bigQueryService;
+  private final UserDao userDao;
+  private final Provider<WorkbenchConfig> workbenchConfigProvider;
+
+  @Autowired
+  AuditController(
+      Clock clock,
+      BigQueryService bigQueryService,
+      UserDao userDao,
+      Provider<WorkbenchConfig> workbenchConfigProvider) {
+    this.clock = clock;
+    this.bigQueryService = bigQueryService;
+    this.userDao = userDao;
+    this.workbenchConfigProvider = workbenchConfigProvider;
+  }
+
+  @VisibleForTesting
+  static String auditTableSuffix(Instant now, int daysAgo) {
+    Instant target = now.minus(daysAgo, ChronoUnit.DAYS);
+    return auditTableNameDateFormatter.withZone(ZoneId.of("UTC")).format(target);
+  }
+
+  private static String auditSql(String cdrProjectId, List<String> tableSuffixes) {
+    // "jobInsertResponse" appears to always be included, despite whether or not job request
+    // metadata was included (i.e. for jobs running in other projects).
+    String tableWildcard = String.format(
+        "%s.%s.cloudaudit_googleapis_com_data_access_*", cdrProjectId, AUDIT_SINK_NAME);
+    return String.format(
+        "SELECT\n" +
+        "  protopayload_auditlog.servicedata_v1_bigquery.jobInsertResponse.resource.jobName.projectId client_project_id,\n" +
+        "  protopayload_auditlog.authenticationInfo.principalEmail user_email,\n" +
+        "  SUM(1) total\n" +
+        "FROM `%s`\n" +
+        "WHERE protopayload_auditlog.methodName = 'jobservice.insert'\n" +
+        "  AND _TABLE_SUFFIX IN (%s)\n" +
+        "GROUP BY 1, 2",
+        tableWildcard,
+        tableSuffixes.stream().map(s -> "'" + s + "'").collect(Collectors.joining(",")));
+  }
+
+  @AuthorityRequired({Authority.AUDIT})
+  @Override
+  public ResponseEntity<AuditBigQueryResponse> auditBigQuery() {
+    // We expect to only see queries run within Firecloud AoU projects, or for administrative
+    // purposes within the CDR project itself.
+    String cdrProjectId = workbenchConfigProvider.get().server.projectId;
+    Set<String> whitelist = Sets.union(userDao.getAllUserProjects(), ImmutableSet.of(cdrProjectId));
```

Comment:
Note: if we ever actually deleted User rows or modified their free tier projects, this would no longer return the projects they had before.

I don't think we're going to do either of those things, so this is fine... just pointing it out.